hazelcast/src/main/java/com/hazelcast/config/LoginModuleConfig.java

```diff
@@ -90,9 +94,13 @@ public LoginModuleConfig setClassName(String className) {
         return this;
     }
 
+    @Deprecated
+    /**
+     * @deprecated Not supported, to be removed in 4.0
+     * @since 3.9

```

Comment:
Any alternative, what to use when I see the deprecation? Maybe in the javadoc and exception message?

Reply:
Apart from that LGTM